src/hooks/screen/useChatScreenHook.ts

```diff
@@ -57,13 +81,15 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       return;
     }
 
+    const replyData = replyPreview;
+    let json;
     let currentChatSchema = sendingChatSchema;
     let userId = await getUserId();
     const myAnonymousId = await getAnonymousUserId();
 
-    if (type === 'ANONYMOUS') {
-      userId = myAnonymousId;
-    }
+    if (type === ANONYMOUS) userId = myAnonymousId;
+    if (replyData) json = {reply_data: replyData};
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The handling of reply data and user identification has been updated. It appears that the changes allow for replies in the chat and anonymous messaging. Make sure that the logic correctly handles these cases.
```diff
+ const replyData = replyPreview;
+ let json;
+ let currentChatSchema = sendingChatSchema;
+ let userId = await getUserId();
+ const myAnonymousId = await getAnonymousUserId();
+ 
+ if (type === ANONYMOUS) userId = myAnonymousId;
+ if (replyData) json = {reply_data: replyData};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->